test/functional/tests/conftest.py

```diff
@@ -180,3 +181,24 @@ def base_prepare(item):
         TestRun.LOGGER.add_build_info(f"{git.get_current_commit_hash()}")
         TestRun.LOGGER.add_build_info(f'Commit message:')
         TestRun.LOGGER.add_build_info(f'{git.get_current_commit_message()}')
+
+
+def add_marks(config):
+    config.addinivalue_line(
+        "markers",
+        "require_compatible_kernel: require minimum two compatible kernel versions"
+    )
+    config.addinivalue_line(
+        "markers",
+        "require_incompatible_kernel: require minimum two incompatible kernel versions"
+    )
```

Comment:
This also isn't CAS specific and can be moved to TF

Reply:
> Robert Baldyga
Those should probably be moved to OCL repository. TF doesn't provide any functionalities related to kernel version, so TF user cannot benefit from these marks in any way.

in comment to closed PR: https://github.com/Open-CAS/test-framework/pull/145/files